static/js/settings_config.ts

```diff
@@ -496,6 +496,11 @@ export const display_settings_labels = {
             defaultMessage: "Convert emoticons before sending (<code>:)</code> becomes 😃)",
         }),
     ),
+    display_emoji_reaction_users: new Handlebars.SafeString(
+        $t_html({
+            defaultMessage: "When possible, show names instead of reaction counts",
```

Comment:
@alya the best I've been able to come up with for this string is this:

"Display names of reacting users when few users have reacted to a message."